client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.ts

```diff
@@ -323,7 +323,24 @@ export class ParticipantDetailViewComponent extends BaseMeetingComponent {
                           .filter(id => !!id)
                     : []
             };
-            await this.repo.update(payload, this.user!).concat(this.repo.setPresent(isPresent, this.user!)).resolve();
+            const title = this.translate.instant(`This action will remove you from one or more groups.`);
```

Comment:
Translation seems to be done in prompt component. So using marker function should be enough here.
```suggestion
            const title = _(`This action will remove you from one or more groups.`);
```